### PR TITLE
DM-20529: Container deployment in Chile with SAL 3.9

### PR DESCRIPTION
**Pull request by tribeiro**
This PR includes a series of changes to re-deploy the containers in Chile with fixed IP numbers and also includes EFD writers and a mariadb.

### CODE REVIEW

File: AT_EFD/docker-compose.yaml [hunk 1]
```diff
@@ -1,103 +1,154 @@
...
-                        #- ${PWD}/mariadb:/var/lib/mysql
+                volumes:
+                        - ${HOME}/efd/mysql:/var/lib/mysql
+                networks:
+                        control-network:
+                                ipv4_address: "192.168.1.2"
+
         EFD_ATDome:
-                image: lsstts/efd_writers
+                image: lsstts/efd_writers:DM-20529
```
Comment (womullan) [inline AT_EFD/docker-compose.yaml:25]: a tag version ?

File: users_jupyter_servers/diag01_setup.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+setup ts_xml -t current
+setup ts_sal -t current
+setup ts_salobj -t current
+setup ts_scriptqueue -t current
+
+echo "#"
+echo "# Starting jupyter lab server"
+
+jupyter lab --ip 192.168.1.201 --port 8885 --no-browser
```
Comment (womullan) [inline users_jupyter_servers/diag01_setup.sh:24]: i thought we now had this in its own container not per host per user which this seems to suggest ..

File: AT_EFD/docker-compose.yaml [hunk 3]
```diff
@@ -1,103 +1,154 @@
...
+                networks:
+                        control-network:
+                                ipv4_address: "192.168.1.9"
                 volumes:
                         - ${OSPL_CONFIG_PATH}:${OSPL_MOUNT_POINT}
                         - ./efd_writers.sh:/home/saluser/efd_writers.sh
                 entrypoint: ["/home/saluser/efd_writers.sh" , "ATPneumatics" , "efdwriter"]
         EFD_ATAOS:
-                image: lsstts/efd_writers
+                image: lsstts/efd_writers:20190514
```
Comment (womullan) [inline AT_EFD/docker-compose.yaml:100]: would some version tag be better -- I assume this is a date ?

**Review by womullan — APPROVED** [review]
ok in general though with fixed ips does that mean we can not spin these up on a local machine for testing .. ?

File: users_jupyter_servers/diag01_setup.sh (hunk 2 above)
Comment (tribeiro) [reply users_jupyter_servers/diag01_setup.sh:24]: not sure I understood the comments. We spin a separate Jupyter server for each user. The idea is to be able to automate the process eventually.

File: AT_EFD/docker-compose.yaml (hunk 1 above)
Comment (tribeiro) [reply AT_EFD/docker-compose.yaml:25]: Yes, I'm still working on getting a tag together for this.

File: AT_EFD/docker-compose.yaml (hunk 3 above)
Comment (tribeiro) [reply AT_EFD/docker-compose.yaml:100]: yes... I still need to work on a better tagging system. this is the build date of the container.

**Pull request merged**